__tests__/app.test.js

```diff
@@ -373,3 +373,33 @@ describe('/api/comments/:comment_id', () => {
   });
 
 });
+
+
+
+describe('/api/users', () => {
+  
+  test('GET 200: Responds with list of users', () => {
+    return request(app)
+    .get('/api/users')
+    .expect(200)
+    .then(({body}) => {
+      const {users} = body
+      users.forEach((user) => {
```

Comment:
When using a forEach in a test, it is vital to check the array has a length greater than 0 (or check the expected length) otherwise we will auto pass all the assertions within the forEach if the array is empty